pwiz_tools/Skyline/app.config

```diff
@@ -672,9 +678,6 @@
             <setting name="AreaCVMsLevel" serializeAs="String">
                 <value>products</value>
             </setting>
-            <setting name="AreaCVRatioIndex" serializeAs="String">

```

Comment:
I also asked Nick to revert all but the expected changes to this file until we can orchestrate everyone upgrading Visual Studio, and making a set of changes like this early in the next release cycle.

Reply:
Nick says we've just gotten out of synch and it should be okay to update this.